Sources/UIExtensions/ViewModifiers/ShakeGestureViewModifier.swift

```diff
@@ -2,6 +2,7 @@
 
 import SwiftUI
 
+#if os(iOS)
```

Comment:
Could you replace above part as below? We don't even need explicit import.
`#if canImport(SwiftUI) && canImport(UIKit)`